apps/www/registry/default/ui/breadcrumb.tsx

```diff
@@ -102,7 +96,7 @@ const BreadcrumbEllipsis = ({
     <span className="sr-only">More</span>
   </span>
 )
-BreadcrumbEllipsis.displayName = "BreadcrumbElipssis"
+BreadcrumbEllipsis.displayName = "BreadcrumbEllipsis"
```

Comment:
This spelling mistake was present in both styles. Not sure if it was intentional, but I went ahead and corrected it on the off-chance it wasn't.